### PR TITLE
Added the Introduction to the Coding Interview Take Home Projects

### LINKED ISSUE #35550: Fill in introduction sections
Some introduction sections still haven't been created.

**PR Created**
- [x] [Intermediate Algorithm Scripting](https://www.freecodecamp.rocks/learn/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/) - @thecodingaviator #34405 
- [x] [Information Security and Quality Assurance Projects](https://www.freecodecamp.rocks/learn/information-security-and-quality-assurance/information-security-and-quality-assurance-projects/) - @Sumer16 #35576 
- [x] [Take Home Projects](https://www.freecodecamp.rocks/learn/coding-interview-prep/take-home-projects/)

**PR Merged**
- [x] [Intermediate Algorithm Scripting](https://www.freecodecamp.rocks/learn/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/)
- [x] [Information Security and Quality Assurance Projects](https://www.freecodecamp.rocks/learn/information-security-and-quality-assurance/information-security-and-quality-assurance-projects/)
- [ ] [Take Home Projects](https://www.freecodecamp.rocks/learn/coding-interview-prep/take-home-projects/)

### PR DESCRIPTION
**Pull request by Sumer16**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #35550

### CODE REVIEW

File: client/src/pages/learn/coding-interview-prep/take-home-projects/index.md [hunk 1]
```diff
@@ -5,6 +5,13 @@ superBlock: Coding Interview Prep
 ---
 ## Introduction to the Coding Interview Take Home Projects
 
-This introduction is a stub
+Programming interviews have always been stressful. One of the solutions is giving the applicant a take home project which they can complete on their own time. This type of interviews usually requires a lot of work but it is considered to be the most preferred interviewing process.
```
Comment (thecodingaviator) [inline client/src/pages/learn/coding-interview-prep/take-home-projects/index.md]:
```suggestion
Programming interviews have always been stressful. One of the solutions is giving the applicant a take home project which they can complete in their own time. These type of interviews usually require a lot of work but it is the most preferred interviewing process.
```

File: client/src/pages/learn/coding-interview-prep/take-home-projects/index.md [hunk 2]
```diff
@@ -5,6 +5,13 @@ superBlock: Coding Interview Prep
...
 
-Help us make it real on [GitHub](https://github.com/freeCodeCamp/learn/tree/master/src/introductions).
\ No newline at end of file
+The interview after project submission could be a discussion about applicant’s design decisions, scalability, and what they might have done out of the box. This approach is more effective when the applicant doesn’t have a lot of publicly available past projects to talk about.
+
+In this section you get the chance to work on many Take Home Projects like:
+*	Show Local Weather
```
Comment (thecodingaviator) [inline client/src/pages/learn/coding-interview-prep/take-home-projects/index.md]: Change this from tab indentation to 2 spaces

**Comment by Sumer16** [comment]
Is this okay now?

File: client/src/pages/learn/coding-interview-prep/take-home-projects/index.md [hunk 3]
```diff
@@ -5,6 +5,13 @@ superBlock: Coding Interview Prep
...
-Help us make it real on [GitHub](https://github.com/freeCodeCamp/learn/tree/master/src/introductions).
\ No newline at end of file
+The interview after project submission could be a discussion about applicant’s design decisions, scalability, and what they might have done out of the box. This approach is more effective when the applicant doesn’t have a lot of publicly available past projects to talk about.
+
+In this section you get the chance to work on many Take Home Projects like:
+*  Show Local Weather
+*  Build a Wikipedia Viewer
+*  Build a Recipe Box and many more...
+
+By the end of this section, you will have 20 Take Home Projects under your belt. This will boost your confidence and make you prepared for Take Home Project Interviews. Have fun and remember to use the [Read-Search-Ask](https://www.freecodecamp.org/forum/t/how-to-get-help-when-you-are-stuck/19514) method if you get stuck.
```
Comment (thecodingaviator) [inline client/src/pages/learn/coding-interview-prep/take-home-projects/index.md]:
```suggestion
By the end of this section, you will have 20 Take Home Projects under your belt. This will boost your confidence and prepare you for Take Home Project Interviews. Have fun and remember to use the [Read-Search-Ask](https://www.freecodecamp.org/forum/t/how-to-get-help-when-you-are-stuck/19514) method if you get stuck.
```

File: client/src/pages/learn/coding-interview-prep/take-home-projects/index.md [hunk 4]
```diff
@@ -5,6 +5,13 @@ superBlock: Coding Interview Prep
...
-Help us make it real on [GitHub](https://github.com/freeCodeCamp/learn/tree/master/src/introductions).
\ No newline at end of file
+The interview after project submission could be a discussion about applicant’s design decisions, scalability, and what they might have done out of the box. This approach is more effective when the applicant doesn’t have a lot of publicly available past projects to talk about.
+
+In this section you get the chance to work on many Take Home Projects like:
+*  Show Local Weather
+*  Build a Wikipedia Viewer
+*  Build a Recipe Box and many more...
+
+By the end of this section, you will have 20 Take Home Projects under your belt. This will boost your confidence and prepare you for Take Home Project Interviews. Have fun and remember to use the [Read-Search-Ask](https://www.freecodecamp.org/forum/t/how-to-get-help-when-you-are-stuck/19514) method if you get stuck.
```
Comment (thecodingaviator) [inline client/src/pages/learn/coding-interview-prep/take-home-projects/index.md]: We'll need to discuss if we want to list the umber of projects over here.

cc: @RandellDawson / @moT01

Comment (moT01) [reply client/src/pages/learn/coding-interview-prep/take-home-projects/index.md]: It says there's 20 projects - I don't think we need to say that twice.

Comment (thecodingaviator) [reply client/src/pages/learn/coding-interview-prep/take-home-projects/index.md]: I want to get the number removed completely. See https://github.com/freeCodeCamp/freeCodeCamp/pull/34405#discussion_r264282065

Comment (thecodingaviator) [reply client/src/pages/learn/coding-interview-prep/take-home-projects/index.md]: Same for here. Reword it to not say no of challenges

**Comment by RandellDawson** [comment]
@Sumer16, @thecodingaviator There is already an earlier/outstanding PR for this file (#35215), which should be reviewed/merged first.

File: client/src/pages/learn/coding-interview-prep/take-home-projects/index.md (hunk 4 above)
Comment (Sumer16) [reply client/src/pages/learn/coding-interview-prep/take-home-projects/index.md]: So what changes needs to be done? @thecodingaviator

**Comment by Sumer16** [comment]
Is this good? @thecodingaviator

File: client/src/pages/learn/coding-interview-prep/take-home-projects/index.md (hunk 4 above)
Comment (thecodingaviator) [reply client/src/pages/learn/coding-interview-prep/take-home-projects/index.md]: Reword it to remove no of challenges. Wait till the PR before this is merged

Comment (Sumer16) [reply client/src/pages/learn/coding-interview-prep/take-home-projects/index.md]: I have changed it @thecodingaviator

File: client/src/pages/learn/coding-interview-prep/take-home-projects/index.md [hunk 5]
```diff
@@ -5,6 +5,13 @@ superBlock: Coding Interview Prep
 ---
 ## Introduction to the Coding Interview Take Home Projects
 
-When applying for a job as a developer, there's a good chance the hiring team will give you something called a "take-home project". As the name suggests, it's something that you're supposed to do on your own, without the interviewer looking over your shoulder. Unlike other parts of the coding interview, which often focus more on theoretical knowledge and less on actual app development - a take-home project is usually the best place to show your skills in practice.
+Programming interviews have always been stressful. One of the solutions is giving the applicant a take home project which they can complete in their own time. These type of interviews usually require a lot of work but it is the most preferred interviewing process.
 
-In this section you'll be able to work on examples of such projects!
\ No newline at end of file
+The interview after project submission could be a discussion about applicant’s design decisions, scalability, and what they might have done out of the box. This approach is more effective when the applicant doesn’t have a lot of publicly available past projects to talk about.
```
Comment (RandellDawson) [inline client/src/pages/learn/coding-interview-prep/take-home-projects/index.md]: It should be `the applicant's design decisions,`  Again, you can not use the word `they` to refer to a single applicant, so `they` should not be used here.  Also, you never end a sentence with a preposition (i.e `about`).  Lastely, "out of the box" is an expression which may confused readers not familiar with such expressions.  Consider rephrasing it in "plain" English.  I don't have a particular suggestion off the top of my head, but I am confident you will think of something.  Note that my suggestion below does not add an alternative for you, but does correct grammar and spelling mistakes.
```suggestion
The interview after the project submission could be a discussion about the applicant’s design decisions, scalability, and what the applicant might have done out of the box. This approach is more effective when the applicant doesn’t have a lot of publicly available past projects.
```

**Review by RandellDawson — CHANGES_REQUESTED** [review]
@Sumer16 Please see my comments regarding several grammar errors.

File: client/src/pages/learn/coding-interview-prep/take-home-projects/index.md (hunk 5 above)
Comment (Sumer16) [reply client/src/pages/learn/coding-interview-prep/take-home-projects/index.md]: I'm sorry for my bad grammar! Thank You for the suggestions. @RandellDawson

**Comment by Sumer16** [comment]
Can you check this one. @RandellDawson

**Review by RandellDawson — APPROVED** [review]

File: client/src/pages/learn/coding-interview-prep/take-home-projects/index.md (hunk 5 above)
Comment (RandellDawson) [reply client/src/pages/learn/coding-interview-prep/take-home-projects/index.md]: Looks good now.

File: client/src/pages/learn/coding-interview-prep/take-home-projects/index.md [hunk 6]
```diff
@@ -5,6 +5,13 @@ superBlock: Coding Interview Prep
 ---
 ## Introduction to the Coding Interview Take Home Projects
 
-When applying for a job as a developer, there's a good chance the hiring team will give you something called a "take-home project". As the name suggests, it's something that you're supposed to do on your own, without the interviewer looking over your shoulder. Unlike other parts of the coding interview, which often focus more on theoretical knowledge and less on actual app development - a take-home project is usually the best place to show your skills in practice.
+Programming interviews have always been stressful. One of the solutions is giving the applicant a take home project to be completed outside of the interview. These types of interviews usually require a lot of work, but it is the most preferred interviewing process.
```
Comment (cmccormack) [inline client/src/pages/learn/coding-interview-prep/take-home-projects/index.md]: Most preferred interviewing process for whom?  The interviewer or interviewee?   Can you provide evidence for this? The previous verbiage was simpler to understand and meant something different.

File: client/src/pages/learn/coding-interview-prep/take-home-projects/index.md [hunk 7]
```diff
@@ -5,6 +5,13 @@ superBlock: Coding Interview Prep
...
 
-In this section you'll be able to work on examples of such projects!
\ No newline at end of file
+The interview after the project submission could be a discussion about the applicant’s design decisions, scalability, and what the applicant might have done differently. This approach is more effective when the applicant doesn’t have a lot of publicly available past projects.
+
+In this section you get the chance to work on many Take Home Projects like:
+*  Show Local Weather
+*  Build a Wikipedia Viewer
+*  Build a Recipe Box and many more...
```
Comment (cmccormack) [inline client/src/pages/learn/coding-interview-prep/take-home-projects/index.md]: Each bullet could be punched up a bit, maybe something like

> * Show Local Weather using location and weather APIs

and many more... can either be removed, or maybe moved to the next line outside the bullets like
> and many more!

File: client/src/pages/learn/coding-interview-prep/take-home-projects/index.md [hunk 8]
```diff
@@ -5,6 +5,13 @@ superBlock: Coding Interview Prep
...
+
+By the end of this section, you will have plenty of Take Home Projects under your belt. This will boost your confidence and prepare you for Take Home Project Interviews. Have fun and remember to use the [Read-Search-Ask](https://www.freecodecamp.org/forum/t/how-to-get-help-when-you-are-stuck/19514) method if you get stuck.
```
Comment (cmccormack) [inline client/src/pages/learn/coding-interview-prep/take-home-projects/index.md]: I believe `take-home` with the hyphen is the proper way to write it.

File: client/src/pages/learn/coding-interview-prep/take-home-projects/index.md (hunk 6 above)
Comment (Sumer16) [reply client/src/pages/learn/coding-interview-prep/take-home-projects/index.md]: For both the Applicant and Technical HR @cmccormack

Comment (Manish-Giri) [reply client/src/pages/learn/coding-interview-prep/take-home-projects/index.md]: @Sumer16 As @cmccormack mentioned, we need a reference linked which states this - 

> but it is the most preferred interviewing process.

**Review by Manish-Giri — CHANGES_REQUESTED** [review]

File: client/src/pages/learn/coding-interview-prep/take-home-projects/index.md (hunk 8 above)
Comment (Sumer16) [reply client/src/pages/learn/coding-interview-prep/take-home-projects/index.md]: I think if we keep it similar to the section heading like "Take Home" it would look better than the hyphen.

Comment (cmccormack) [reply client/src/pages/learn/coding-interview-prep/take-home-projects/index.md]: Fix the header instead, then it will all be accurate.

**Comment by RandellDawson** [comment]
@Sumer16 You still have not addressed @Manish-Giri's concern about providing a reference link regarding your statement:
> but it is the most preferred interviewing process for the interviewers.

Who says this is the most preferred interview process for the interviewers?  Has there been a published study?  If so, please provide the documentation or remove this entire sentence.

**Comment by Sumer16** [comment]
This is an Article on Hacker Noon- [How Interview Projects Stop Bad Hires](https://hackernoon.com/how-interview-projects-stop-bad-hires-b280bef441e7)

[What the Hiring Manager's Actually Looking For in That Take-Home Assignment](https://www.themuse.com/advice/heres-what-the-hiring-managers-actually-looking-for-in-that-dreaded-takehome-assignment)

It seems that it allows the Interviewers to look over the interviewee's skills for the company.

**Comment by moT01** [comment]
I couldn't find anything in those links that says it's the most preferred process @Sumer16. So I would just remove that line to satisfy the requests so we can get this merged. Or, you could maybe change it to say something like... "These types of interviews usually require a lot of work, but they're a great way for employers to see how you might perform in a real job scenario."

**Comment by moT01** [comment]
I committed a change to this to hopefully get it ready to go. Want to take another look and merge it if you're satisfied @Manish-Giri?

**Review by Manish-Giri — APPROVED** [review]
@moT01 Thanks! Changes LGTM.

**Comment by Sumer16** [comment]
Thank You @Manish-Giri for accepting my Pull Request.

**Comment by Sumer16** [comment]
Thank You! @moT01 for the changes.
